resources/assets/less/bem/mp-history-game.less

```diff
@@ -0,0 +1,216 @@
+/**
+ *    Copyright 2016 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+.mp-history-game {
+  width: 100%;
+  .default-border-radius;
+
+  margin-bottom: @spacing * 2;
+
+  &__header {
+    width: 100%;
+    height: 100px;
+    .at2x-simple('/images/layout/beatmaps/default-bg.png');
+
+    border-radius: @border-radius-base @border-radius-base 0 0;
+  }
+
+  &__header-overlay {
+    width: 100%;
+    height: 100%;
+
+    background-color: #000000;
+    opacity: 0.5;
+
+    border-radius: @border-radius-base @border-radius-base 0 0;
+  }
+
+  &__thumb {
+    background-position: center;
+    background-size: cover;
+    display: block;
+    height: 100px;
+
+    border-radius: @border-radius-base @border-radius-base 0 0;
+  }
+
+  &__stats-box {
+    display: flex;
+
+    position: absolute;
+    left: 10px;
+    top: 10px;
+  }
+
+  &__stat {
+    font-size: 14px;
+    font-style: italic;
+
+    padding-right: @spacing * 1.5;
+
+    color: #ffffff;
+
+    .default-text-shadow;
+  }
+
+  &__metadata-box {
+    position: absolute;
+    left: 10px;
+    bottom: 10px;
+  }
+
+  &__metadata {
+    font-style: italic;
+    color: #ffffff;
+    .default-text-shadow;
+
+    margin: 0;
+
+    &--title {
+      font-size: 20px;
+    }
+
+    &--artist {
+      font-size: 14px;
+    }
+  }
+
+  &__mods-box {
+    position: absolute;
+    right: 10px;
+    top: 10px;
+  }
+
+  &__team-type {
+    position: absolute;
+
+    right: 10px;
+    bottom: 10px;
+
+    width: 40px;
+    height: 40px;
+
+    background-size: 40px 40px;
+  }
+
+  &__player-scores {
+    padding: 10px;
+    background-color: @grey-e;
+
+    &--no-teams {
+      border-radius: 0 0 @border-radius-base @border-radius-base;
+    }
+  }
+
+  &__player-score {
+    width: 100%;
+    height: 60px;
```

Comment:
i don't get where are you going with this, these elements are fixed height because they had those dimensions in the designs.
also how am i supposed to replace height with padding-bottom?